languages/python/exercises/concept/dicts/.docs/instructions.md

```diff
@@ -0,0 +1,48 @@
+In this exercise you'll be managing an inventory system.
+
+You will be given a list of items. Each time an item is in the given list, add `1` to the key in the _given_ inventory. Each item should be organized by their name and the amount of that item. You will also have to delete items from a dictionary.
+
+You will also have to implement a function which returns a list of `tuples` of all the key-value pairs in the _given_ inventory.
+
+## 1. Create an inventory from a list
+
+Implement a function that creates an "inventory" from a list of items. It should return a `dictionary` representing the types and amounts of the items.
+
+```python
+>>> create_inventory(["coal", "wood", "wood", "diamond", "diamond", "diamond"])
+{"coal":1, "wood":2 "diamond":3}
+```
+
+## 2. Add items from a list to an existing dictionary
+
+Implement a function that adds a list of items to a passed in inventory dictionary:
+
+```python
+>>> add_items({"coal":1}, ["wood", "iron", "coal", "wood"])
+{"coal":2, "wood":2, "iron":1}
+```
+
+## 3. Remove items from an existing inventory dictionary
```

Comment:
The "dictionary" bit is implementation-specific, whereas we'd like task headings to be more about what to do instead of how. Maybe do:

```suggestion
## 3. Remove items from the inventory
```